pkgs/leak_tracker_testing/lib/src/model.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Object lifecycle.
+///
+/// Is used to test if an object dispatched deeded events at creation and disposal.
+class ObjectLyfecycle {
```

Comment:
Should this be `ObjectLifecycle`?

Reply:
I little redesigned the matcher and the type is not needed any more.